pkg/terraform/executor.go

```diff
@@ -516,31 +516,31 @@ func (ex *Executor) logPath(id int) string {
 func (ex *Executor) checkVersion() error {
 	vOutput, err := ex.executeSync("--version")
 	if err != nil {
-		return fmt.Errorf("Error checking Terraform version: %w", err)
+		return fmt.Errorf("error checking Terraform version: %w", err)
 	}
 
 	var vStr string
 	n, err := fmt.Sscanf(string(vOutput), "Terraform v%s\n", &vStr)
 	if err != nil {
-		return fmt.Errorf("Error checking Terraform version: %w", err)
+		return fmt.Errorf("error checking Terraform version: %w", err)
 	}
 
 	if n != 1 {
-		return fmt.Errorf("Parsing Terraform version failed")
+		return fmt.Errorf("parsing Terraform version failed")
```

Comment:
Are you sure about this one? Omitting 'failed' ?
`parsing Terraform version` will look like a log message over an error message.
Maybe we can go for `error while parsing Terraform version` ? @invidian

Reply:
@invidian now the string `parsing Terraform version` will look like a log message instead of an error message.
how about `error occured while parsing Terraform version` or `unable to parse Terraform version`?

Reply:
this might help. Please check https://github.com/kinvolk/lokomotive/pull/953/files#diff-a4a1697930a32405cd5efdc7aa09b71fR38

Reply:
Hmm, now that I think about it, this error message doesn't seem helpful at all. We could include expected format and parsed value in it.

Also, I look at the documentation of `fmt.Sscanf` right now and try to figure out in which case it returns error here.

Looking at https://golang.org/src/fmt/scan.go?s=4522:4597#L1181, I _think_ it will always return err if `n` is different than number of operands passed. Would be good to have some unit tests for it :/